pyproject.toml

```diff
@@ -54,6 +54,7 @@ langchain-community = "^0.2.7"
 ijson = "^3.3.0"
 psutil = "^6.0.0"
 langchain-aws = "^0.1.11"
+wonderwords = "^2.2.0"
```

Comment:
As discussed, if we can stick with faker we can drop this!

Reply:
Now removed.